pl/lessons/basics/comprehensions.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: page
+title: Odwzorowania kolekcji
+category: basics
+order: 13
+lang: pl
+---
+
+Odwzorowania kolekcji, zwane też wyrażeniami generatorowymi (ang. list comprehensions), są rozszerzeniem składni
```

Comment:
> Odwzorowania kolekcji, zwane też wyrażeniami generatorowymi (ang. list comprehensions), są rozszerzeniem składni
 +języka Elixir pozwalającym na wygodniejszą pracę z kolekcjami i danymi przeliczalnymi.

What about?

> Składanie list, to lukier składniowy pozwalający na wygodniejszą pracę z kolekcjami i danymi przeliczalnymi.

I suggest here to use translation based on Python language - I thinks it's one of the first to introduce this feature.